Package.swift

```diff
@@ -6,8 +6,7 @@ import PackageDescription
 let package = Package(
     name: "Siwf",
     platforms: [
-        .iOS(.v15),
-        .macOS(.v11)
+        .iOS(.v15)
```

Comment:
So the deployment broke mac?

Reply:
Yeah. I worked on making it be a part of it, but it was one thing after another. On top of that I couldn't find a way to make macOS support the callback part of it at all.